translate/filters/checks.py

```diff
@@ -2334,6 +2334,28 @@ def __init__(self, **kwargs):
         StandardChecker.__init__(self, **kwargs)
 
 
+reducedconfig = CheckerConfig()
+
+
+class ReducedChecker(StandardChecker):
+
+    def __init__(self, **kwargs):
+        checkerconfig = kwargs.get("checkerconfig", None)
+
+        if checkerconfig is None:
+            checkerconfig = CheckerConfig()
+            kwargs["checkerconfig"] = checkerconfig
+
+        limitfilters = kwargs.get("limitfilters", None)
+
+        if limitfilters is None:
+            limitfilters = ["untranslated", "unchanged", "blank", "doublespacing", "doublewords", "spellcheck"]
```

Comment:
I'm pretty sure that startcaps and endcaps woud generate a lot of false positives in my case.